services/authentication.py

```diff
@@ -163,7 +163,7 @@ def verify_code(session: Session, email: str, code: str):
     #  We can add captcha to make it safer
 
     @staticmethod
-    def reset_password(session: Session, email: str, new_password: str, repeat_password: str):
+    def reset_password_with_email(session: Session, email: str, new_password: str, repeat_password: str):
```

Comment:
Get rid of this bit. Just leave it as is

Reply:
We can't securely allow password reset without email. If its broken, its broken, but at least its secure